src/db/tables/subscribers.js

```diff
@@ -8,6 +8,7 @@ import AppConstants from '../../appConstants.js'
 
 const knex = createDbConnection();
 const { DELETE_UNVERIFIED_SUBSCRIBERS_TIMER } = AppConstants
+const MONITOR_PREMIUM_CAPABILITY = "monitor";
```

Comment:
Should this be in `constants.ts`, since it's used in more than one place?

Reply:
Oh right, I was actually going to ask you, since I copied this pattern [from](https://github.com/mozilla/blurts-server/commit/3bf2291258c100b1d8097ba5701276cb49e9533c#diff-97bf25d3506c6da45f409e35989ed5b2c1de1f9aed63ae4f26cbfea60c7c8e55R8) [you](https://github.com/mozilla/blurts-server/commit/e80edf72644811bbc8766d2023bcd914a50393bf#diff-836cc645d4daafe3034b9a12b669a35290f6006abf29e4565715a6593ff8ae54R32) :sweat_smile: